templates/admin/edit-page.html.twig

```diff
@@ -0,0 +1,27 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}
+Édition de la page statique : {{ page.title }}
+{% endblock %}
+
+{% block main_content %}
+    {% include 'components/app-flashes.html.twig' %}
```

Comment:
app-flashes are now loaded in base.html.twig

Reply:
Oh nice!